### PR TITLE
Basic Translation Infrastructure

### PR DESCRIPTION
**Pull request by sdreher**
* user selects language at registration or updates in profile
* language is set on login / or when profile is updated
* language settings / locale files, etc.
* i18n urls
* pagetree hierarchy selected based on view/edit urls

### CODE REVIEW

File: nepi/settings_production.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
 if 'migrate' not in sys.argv:
     INSTALLED_APPS.append('raven.contrib.django.raven_compat')
 
+LOCALE_PATHS = ('/var/www/nepi/meaningfulconsent/locale',)
```
Comment (thraxil) [inline nepi/settings_production.py]: Not being terribly familiar with this project, what is the relationship between NEPI and meaningfulconsent, exactly?

Comment (sdreher) [inline nepi/settings_production.py]: meaningfulconsent is what I copied from. :-) I'll update.

Comment (thraxil) [inline nepi/settings_production.py]: We probably ought to be better about doing relative paths for these kinds of settings  anyway. Ie, `os.path.join(os.path.dirname(__file__), 'locale')`

Comment (sdreher) [inline nepi/settings_production.py]: agreed.

**Comment by sdreher** [comment]
Updated paths based on the ccnmtldjango example. Does this structure seem correct?

**Pull request merged**